<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Test:testA is Fail]]>
    </line>
    <line>
      <![CDATA[TestName:testA Status:FAIL]]>
    </line>
    <line>
      <![CDATA[PHOTO:./photo/Thu Oct 04 12_53_07 IST 2018.png]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="12979" started-at="2018-10-04T07:22:56Z" finished-at="2018-10-04T07:23:09Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12979" started-at="2018-10-04T07:22:56Z" finished-at="2018-10-04T07:23:09Z">
      <class name="script.DemoA">
        <test-method status="PASS" signature="openApp(java.lang.String, java.lang.String)[pri:0, instance:script.DemoA@482f8f11]" name="openApp" is-config="true" duration-ms="8172" started-at="2018-10-04T12:52:56Z" finished-at="2018-10-04T12:53:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[localhost]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method status="FAIL" signature="testA()[pri:0, instance:script.DemoA@482f8f11]" name="testA" duration-ms="3118" started-at="2018-10-04T12:53:04Z" finished-at="2018-10-04T12:53:07Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy12.sendKeys(Unknown Source)
	at page.LoginPage.setUserName(LoginPage.java:23)
	at script.DemoA.testA(DemoA.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Test:testA is Fail]]>
            </line>
          </reporter-output>
        </test-method> <!-- testA -->
        <test-method status="PASS" signature="closeApp(org.testng.ITestResult)[pri:0, instance:script.DemoA@482f8f11]" name="closeApp" is-config="true" duration-ms="1648" started-at="2018-10-04T12:53:07Z" finished-at="2018-10-04T12:53:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testA status=FAILURE method=DemoA.testA()[pri:0, instance:script.DemoA@482f8f11] output=Test:testA is Fail]]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[TestName:testA Status:FAIL]]>
            </line>
            <line>
              <![CDATA[PHOTO:./photo/Thu Oct 04 12_53_07 IST 2018.png]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- script.DemoA -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
